### ISSUE TITLE
Paper Db deserialization Issue On App Update

### ISSUE THREAD

**Issue by Ahssan18** [pilgr/Paper#193]
System.err: io.paperdb.PaperDbException: Couldn't read/deserialize file /data/user/0/com.example.app/files/io.paperdb/company.pt for table company
10-21 16:26:29.294 32293 32293 W System.err: at io.paperdb.DbStoragePlainFile.readTableFile(DbStoragePlainFile.java:323)
10-21 16:26:29.294 32293 32293 W System.err: at io.paperdb.DbStoragePlainFile.select(DbStoragePlainFile.java:166)
10-21 16:26:29.294 32293 32293 W System.err: at io.paperdb.Book.read(Book.java:73)
10-21 16:26:29.294 32293 32293 W System.err: at io.paperdb.Book.read(Book.java:58)

When update app on android 10 and read data from previous paper db table it throughs Exception.

**Comment by tmcintosh** [comment]
This suddenly started occuring to me as well.

I've never had an issue with PaperDb and upgrading releases until now.

No code in my latest release has touched the database implementation. Package name for the only serialized class hasn't changed.  Perhaps something in the build process?  I'm quite confused, lol.

Still on gradle 7.0.1 as 7.0.2 has issues with PaperDb.

**Comment by tmcintosh** [comment]
I'm sorry to say, but I ripped out paperDb from my app and since the newest release I've had literally zero crashes;  It's almost unheard of for me to have zero crashes days after release given the (admittedly small) volume.  I replaced it with my own RoomDb implementation. 

PaperDb was great for early development and prototyping but currently in a production scale it's highly unstable and the support for fixing these issues is lacking.  I wish everyone the best.
